client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -248,12 +248,15 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
 
   private val messagesHelper: TransportMessagesHelper = new TransportMessagesHelper()
 
+  private val extraInfo = ApplicationInfoProvider.instantiate(conf)
+    .provide() + (CelebornConf.QUOTA_INTERRUPT_SHUFFLE_ENABLED.key -> conf.quotaInterruptShuffleEnabled.toString)
+
   private def registerApplicationInfo(): Unit = {
     Utils.tryLogNonFatalError(
       masterClient.send(RegisterApplicationInfo(
         appUniqueId,
         userIdentifier,
-        ApplicationInfoProvider.instantiate(conf).provide().asJava)))
+        extraInfo.asJava)))
```

Comment:
how about move the extraInfo definition into the `tryLogNonFatalError`?

So that, it won't fail the app if the `ApplicationInfoProvider.instantiate` failed.